.node-version

```diff
@@ -0,0 +1 @@
+6
```

Comment:
for posterity, lets update it to `6.11.1` here and in the `.travis.yml`. its nice to be exact